docs/release-notes/2.11.0.asciidoc

```diff
@@ -4,6 +4,11 @@
 [[release-notes-2.11.0]]
 == {n} version 2.11.0
 
+[[breaking-2.11.0]]
+[float]
+=== Breaking changes
+
+* The `resourceStatuses` field of the status subresource of the Stack Configuration Policy has been renamed to `details`. {pull}7433[#7433]
```

Comment:
This seems a little "wordy" to me, so suggestions welcome on whether or not we want this much detail in the release notes.